### DISCUSSION TITLE
Latest Copilot 1.7.4688 claims VSCode is outdated

### DISCUSSION THREAD

**Discussion by Rigel772** [community/community#10403] [Copilot Conversations]
It shows message
```GitHub Copilot requires VS Code 1.57+. Please update your VS Code```
on VSCode 1.62 despite having setting:
```"enable-proposed-api": ["github.copilot"]```
in Configure Runtime Arguments

**Comment by davecheney** [comment]
Are you using VSCodium or OSS Code?

**Reply by Rigel772** [reply]
Code OSS

**Reply by davecheney** [reply]
I’m sorry, Copilot needs VSCode, it does not currently support OSS Code.

**Reply by Zauberbutter** [reply]
Why don't you say what could causes this issue? I mean it worked before with the runtime setting. So maybe its just another setting that must be tweaked.

**Reply by davecheney** [reply]
In all honesty I don’t know why Copilot doesn’t work with those variants. I know that it is being looked into, but I cannot share an eta with you.

**Reply by Krzysztof-Cieslak** [reply]
@Rigel772, please make sure you are using the latest version of Code (1.63.X)

**Reply by Rigel772** [reply]
I switched to VSCode and have no issues since.
